src/portal/src/app/distribution/distribution-instances/distribution-instances.component.ts

```diff
@@ -167,8 +167,10 @@ export class DistributionInstancesComponent implements OnInit, OnDestroy {
       operMessage.state = OperationState.progressing;
       operMessage.data.name = this.selectedRow[0].name;
       this.operationService.publishInfo(operMessage);
+      const instance: Instance = clone(this.selectedRow[0]);
+      instance.default = true;
       this.disService.UpdateInstance({
-          propertySet: {default: true},
+          instance: instance,
```

Comment:
as the instance has name property, when do we still need the second param `preheatInstanceName: this.selectedRow[0].name`?